www/docs/en/6.x/config_ref/images.md

```diff
@@ -135,6 +136,22 @@ different screen resolutions.
         <icon src="res/Windows/Wide310x150Logo.scale-240.png" width="744" height="360" />
     </platform>
 ```
+
```

Comment:
Yes, they are get copied during 'prepare' phase. The resultant filename is calculated based on `width` and `height` attributes using [hardcoded map](https://github.com/apache/cordova-windows/pull/159/files#diff-32bff2189ca335a5581636fff314a504R315). The source filename does not matter in this case and is used only to indicate where to get source images.